### DISCUSSION TITLE
Preview files without caching them before

### DISCUSSION THREAD

**Discussion by vvidalseedbox** [iterate-ch/cyberduck#16208] [Q&A]
Hi Everyone,
Is it possible in Cyberduck to open a multimedia file as in finder, i.e. almost immediately without caching the file before previewing it?
In Finder, when I open a video located in a server it's almost instant when i do the same thing in Cyberduck with the same video it takes much longer, depending on the video it can take up to 10 mins for an 8GB video.
I'm actually using the SMB protocol (not by choice) to connect to this server and before I downgraded to an earlier version of Cyberduck, it took 40 mins to preview a 8 GB video, my question is, is there a way to make the preview instant like on Finder or File Explorer?

**Comment by tatze96** [comment]
It isn't possible to open or preview files from Cyberduck without downloading them beforehand, at least temporarily, as the file contents wouldn't be visible otherwise.
